Annotation/Job.php

```diff
@@ -2,49 +2,54 @@
 
 namespace Padam87\CronBundle\Annotation;
 
+use Attribute;
```

Comment:
Please do not import from the global namespace (there are multiple instances of this, I'll only write it here)

Reply:
check

Reply:
I will remove all lthe non-related code-style changes in this PR. If you want I can create a new PR after this one is merged to update the codebase to PHP 7.4 or higher.

Reply:
Thanks you!